pkg/csi/service/common/common_controller_helper.go

```diff
@@ -141,6 +142,41 @@ func CheckAPI(version string) error {
 	return nil
 }
 
+// UseVslmAPIs checks if specified version is between 6.7.3 and 7.0.0
```

Comment:
Can we add unit tests for this func to quickly validate all different versions of vSphere?

Reply:
Added unit tests in the latest commit
```
$ go test -v
=== RUN   TestUseVslmAPIsFunc
--- PASS: TestUseVslmAPIsFunc (0.00s)
```